Modix.Data.Test/TestData/TagActions.cs

```diff
@@ -132,7 +132,7 @@ public static TagActionEntity Clone(this TagActionEntity entity)
                 Created = entity.Created,
                 Type = entity.Type,
                 CreatedById = entity.CreatedById,
-                CreatedBy = entity.CreatedBy?.Clone(),
+                CreatedBy = entity.CreatedBy?.Clone()!,
```

Comment:
Honestly, I'm not sure what does or does not seem right about this. Which part are you commenting on?

Reply:
Basically, my thinking is...

`CreatedById` cannot be `null`. Both literally, and conceptually. An entity cannot exist without an associated creation action. `CreatedBy` should be annotated as not-`null`able, accordingly.

The literal implementation of `TagActionEntity` requires that `CreatedBy` be initialized with `null` because EF cannot load navigational properties through the constructor. Also, an EF query might not `.Include(x => x.CreatedBy)`.

However, this bit of code is for test-land, specifically for allowing a set of test entities to be used by many tests, without the potential for mutation of these entities to carry-over between tests. Does the set of common test entities include entities where `CreatedBy` is not "loaded"?